modules/rostests/apitests/atl/CAtlArray.cpp

```diff
@@ -9,27 +9,7 @@
 #ifdef HAVE_APITEST
```

Comment:
I would also move this `#ifdef` into the header, but if you prefer to leave it here that is fine too.